src/fileio.jl

```diff
@@ -86,9 +115,11 @@ end
 
 function save(s::Stream{format"LAS"}, header::LasHeader, pointdata::AbstractVector{<:LasPoint})
     # checks
-    header_n = header.records_count
+    header_n = header.extended_number_of_point_records > 0 ? Int(header.extended_number_of_point_records) : Int(header.records_count)
```

Comment:
This line appears often. Maybe a little function `get_record_count(header)` can centralize this?

Reply:
Done!